### ISSUE TITLE
Swipe action to make feed item unaffected from `Mark as read on scroll` for one scroll

### ISSUE THREAD

**Issue by KAGEYAM4** [spacecowboy/Feeder#373]
### Checklist

- [X] I have used the search function for [**OPEN**](https://github.com/spacecowboy/feeder/issues) issues to see if someone else has already submitted the same feature request.
- [X] I have **also** used the search function for [**CLOSED**](https://github.com/spacecowboy/feeder/issues?q=is%3Aissue+is%3Aclosed) issues to see if the feature was already implemented and is just waiting to be released, or if the feature was rejected.
- [X] I will describe the problem with as much detail as possible.
- [X] This request contains only one single feature, **not** a list of multiple (related) features.

### App version

2.6.32

### Where did you get the app from

F-Droid

### Feature or improvement you want

I use `Mark as read on scroll`.  Make a feed unaffected by one-scroll action and thus not be marked as read.

Implementation - For now i can only think of adding an option in context menu of feed item ( i.e beside `save article`).

Failed implementation - 
1. Replace swipe (read/unread) action with `unaffected by one-scroll` but then when we are bottom of feed page `Mark as read on scroll` won't work and only way to mark feed as read at that point is swipe action.
2. Add swipe action to both feed-image and feed-info, but then there are feed items that don't contain feed-image.

### Why should this be added?

While scrolling feeds sometimes i come across feeds which i would like to read-later or perform some task based on it but not wanna deal with it right now. Middle ground between viewed and saved-article.

Why am i not using `saved article`? - For me saved Articles are things which gonna take multiple session to complete or bookmark.

**Comment by spacecowboy** [comment]
To answer your question, I do not use mark as read on scroll.

I periodically press "mark all as read" (checkmark button), or use long press and mark all below as read.

**Comment by KAGEYAM4** [comment]
I noticed when using `Swipe to mark as read: only from end` there is swipe left when swiping from right end. Here's a video -->


https://github.com/user-attachments/assets/c245c42c-d1b9-4477-81dd-f2d8ad9bef57



Can we use this to perform action `Make feed item unaffected from Mark as read on scroll for one scroll-over item` ?

![Screenshot_20241005-135007_Feeder](https://github.com/user-attachments/assets/2b48fc82-520c-4dbd-ab81-96d27c0d88da)

**Comment by Matth7878** [comment]
Agree with that one, not only for mark read on scroll but to ignore all actions marking as read expect actually reading.
So an "ignore marking as read" or "keep unread" or "read later".

If you want to mark several articles for read later using saved articles is cumbersome : 
- main annoyance is when reading it isn't unmarked as saved, if you marked a lot of articles to read later you would have to unsaved them all individually once you read them
- saved articles are marked as read on mark all read, so you have no visual way to know which saved articles you actually read vs one you already read
- beside there is no counter in drawer to know which tags have saved articles

To me saving article is akin to marking as favorite and read later would be like you want to read but can't right now.

**Comment by spacecowboy** [comment]
I'm not sure this feature request is actually doing the right thing.

Save article was built exactly for this reason. Keep an article around for later, until you want to read it.

So in your filters, have you enabled `Show saved articles`? This is what I use to keep articles around. Why is this not a satisfactory solution to you? I think that is the main question to address.

**Comment by Matth7878** [comment]
Concerning me I stated in my previous post why saved articles was not ideal : 

- main annoyance is when reading it isn't unmarked as saved, if you marked a lot of articles to read later you would have to unsaved them all individually once you read them
- saved articles are marked as read on mark all read, so you have no visual way to know which saved articles you actually read vs one you already read
- beside there is no counter in drawer to know which tags have saved articles

Besides saving articles is more something you want to save forever (at least with current implementation) vs something you didn't have time to read and want to later. (But not keep forever)

**Comment by spacecowboy** [comment]
> Besides saving articles is more something you want to save forever (at least with current implementation) vs something you didn't have time to read and want to later. (But not keep forever)

That depends on the user. Personally I only use it for short-term saving.

For stuff I want to be able to find far into the future, I use Mozilla's Pocket app.

Regarding read/unread, for that isn't an issue since I don't save that many (less than 10) articles at once. I remember which I have read or not.

TL;DR; I think the problem statement is a valid one. But I don't think weird workarounds to avoid having articles being marked as read is the way to solve it.

**Comment by KAGEYAM4** [comment]
> TL;DR; I think the problem statement is a valid one. But I don't think weird workarounds to avoid having articles being marked as read is the way to solve it.

I will try asking and maybe trying other popular feed reader to see if they have any good UX for this.

But here some-suggestion i thought of -->
-  How about After user has finished with all the unread feeds in a feed/tag. Saved articles would start showing up. There will be a gap between them with maybe heading.
- Read/Unread dosen't make sense in case of saved articles. It can be there no need to remove it. But in case of saved articles it makes sense to have slide-gesture to remove them. Hold-tap to mark-as-unsaved seems not big but it becomes repeating activity when one wants to go through all of its saved articles. So maybe we can replace read/unread for saved with unsave?
- About the  gesture diagram i showed in https://github.com/spacecowboy/Feeder/issues/373#issuecomment-2394981443, is this possible ? Maybe we use `save/unsave` instead of `unaffected by scroll`.


Maybe we can take a look at other feed reader to see there UX (if they have one) regarding this.
